packages/next/package.json

```diff
@@ -143,7 +143,7 @@
     "@babel/traverse": "7.18.0",
     "@babel/types": "7.18.0",
     "@capsizecss/metrics": "1.1.0",
-    "@edge-runtime/cookies": "3.2.1",
+    "@edge-runtime/cookies": "3.3.0",
```

Comment:
@ztanner I'm wondering if this is still in the change. Looks like it has been removed from the `Files changed` section.

Reply:
Got it. Thank you!